src/test/kotlin/coffeelisp/functions/RandomTest.kt

```diff
@@ -0,0 +1,34 @@
+package coffeelisp.functions
+
+import coffeelisp.env.eval
+import org.junit.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFails
+
+class RandomTest {
+    @Test
+    fun testOverBound() {
+        val maxInt = Int.MAX_VALUE
+        assertFails {
+            "(random (+ $maxInt 1)".eval()
```

Comment:
I think this is missing a right paren, and probably failing for the wrong reason